tests/lib/prepare.sh

```diff
@@ -633,9 +633,7 @@ EOF
     # the writeable-path sync-boot won't work
     mkdir -p /mnt/system-data/etc/systemd
 
-    # we do not need console-conf, so prevent it from running
     mkdir -p /mnt/system-data/var/lib/console-conf
-    touch /mnt/system-data/var/lib/console-conf/complete
```

Comment:
I think this change is fine to do globally, also it seems that we (accidentally probably) never implemented this same disabling of console-conf on uc20.